guides/common/modules/ref_capsule-ports-and-firewalls-requirements.adoc

```diff
@@ -31,50 +32,45 @@ The following tables indicate the destination port and the direction of network
 [cols="24%,20%,18%,38%",options="header"]
 |====
 | Port | Protocol | Service | Required For
+| 443 | TCP | HTTPS | {SmartProxy} to {Project}
+ifdef::katello[]
 | 5646   | TCP   |  amqp   |  {SmartProxy}'s Qpid dispatch router to Qpid dispatch router in {Project}
+endif::[]
 |====
 
 ifdef::foreman-el,katello[]
 Some of the following ports apply only to deployments that use the Katello plug-in.
 endif::[]
 
-ifdef::foreman-deb[]
-Some of the following ports apply only to deployments that use the Katello plug-in.
-For Debian deployments, ignore references to the Katello plug-in.
-endif::[]
-
 .Ports for Client to {SmartProxy} Communication
 [cols="24%,20%,18%,38%",options="header"]
 |====
-ifdef::satellite[]
 |Port |Protocol |Service |Required for
-|80 |TCP |HTTP |Anaconda, yum, and for obtaining Katello certificate
-updates
-|443 |TCP |HTTPS |Anaconda, yum, Telemetry Services, and Puppet
+ifdef::satellite[]
+|80 |TCP |HTTP |Anaconda, yum, and for obtaining Katello certificate updates
+|443 |TCP |HTTPS |Anaconda, yum, and Telemetry Services
 |5647 |TCP |AMQP |Katello agent to communicate with {SmartProxy}'s
 Qpid dispatch router
 |8000 |TCP |HTTPS |Anaconda to download kickstart templates to hosts,
 and for downloading iPXE firmware
+|8443 |TCP |HTTPS |Subscription Management Services and Telemetry Services
 endif::[]
-ifdef::foreman-el,katello[]
-|Port |Protocol |Service |Required for
-|80 |TCP |HTTP |Operating System installers like Anaconda, yum, and, if you use the Katello plug-in, for obtaining Katello certificate
-updates
-|443 |TCP |HTTPS |Operating System installers like Anaconda, yum, Telemetry Services, and Puppet
+ifdef::katello[]
+|80 |TCP |HTTP |Operating System installers like Anaconda, yum, and for obtaining Katello certificate updates
+|443 |TCP |HTTPS |Operating System installers like Anaconda, yum, and Telemetry Services
```

Comment:
```suggestion
|443 |TCP |HTTPS |Operating System installers like Anaconda, content, and telemetry services
```

As stated above, we might should drop Telemetry Services since I dont know what that is.